lib/torrent/scheduler/constructors.go

```diff
@@ -69,10 +76,16 @@ func NewOriginScheduler(
 		stats,
 		pctx,
 		announceclient.Disabled(),
-		announcequeue.Disabled(),
 		netevents)
 	if err != nil {
 		return nil, err
 	}
-	return makeReloadable(s, func() announcequeue.Queue { return announcequeue.Disabled() }), nil
+
+	aq := func() announcequeue.Queue { return announcequeue.Disabled() }
```

Comment:
maybe the constructor can just return a function

Reply:
Eh, I'd rather no complicate the announcequeue package because of the weird details up here.